hack/update-generated-crd.sh

```diff
@@ -10,23 +10,29 @@ function install_crd {
   fi
 }
 
+
 # Can't rely on associative arrays for old Bash versions (e.g. OSX)
-install_crd \
+
+[[ -e "vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_ingress_00_ingresscontrollers-Default.crd.yaml" ]] && install_crd \
   "vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_ingress_00_ingresscontrollers-Default.crd.yaml" \
   "manifests/00-custom-resource-definition.yaml"
```

Comment:
Does this work?  I think you need something like this to match either `0000_50_ingress_00_ingresscontrollers-Default.crd.yaml` or `0000_50_ingress_00_ingresscontrollers.crd.yaml`:
```suggestion
shopt -s extglob
verify_crd \
  vendor/github.com/openshift/api/operator/v1/zz_generated.crd-manifests/0000_50_ingress_00_ingresscontrollers?(-Default).crd.yaml \
  "manifests/00-custom-resource-definition.yaml"
```

Reply:
yes that is a nice way to do pattern match file.

Reply:
done